examples/trace-analytics-sample-app/opentelemetry-collector/otel-collector-dns-config.yml

```diff
@@ -0,0 +1,17 @@
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:55680
+
+exporters:
+  otlp/2:
+    endpoint: prepper-cluster:21890
```

Comment:
QUES: just curious. Will 10.10.2.1:21890 also work?

Reply:
Correct, any of the following will work.
* `10.10.2.1` - the address of the prepper instance
* `data-prepper` - provided by [docker-compose](https://docs.docker.com/compose/networking/) (hostname = container name)
* `prepper-cluster` - defined in dnsmasq configuration

In this example I'm pointing the otel exporter at `prepper-cluster` so we can easily test with multiple hosts by adding entries to the dnsmasq hosts.prepper file.